### PR TITLE
Milestone 7 :- Deployment of the project.

### PR DESCRIPTION
**Pull request by hmunish**
I have added an index and about page for both the mobile & desktop version of the website.
These changes make the user use home & about pages on mobile & desktop both.
This includes:-
1. Linters configuration files (ESLint, StyleLint, LightHouse) to check code typing & convention errors.
2. A README.md file with details about the repository & LICENSE.txt file with license to use the repository.
3. Index & About Page for the website.
4. Responsive Navbar with the dropdown feature using HTML, CSS & JavaScript.
5. Banner section with website details using HTML heading tags. 
6. Feature section with separate features using HTML image tag.
7. Dynamic Speakers section using JavaScript Event Listeners & Objects to render data.
8. Partners section using CSS Flexbox.
9. Footer with copyright information.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+         <section class="mobile menu dsp_none">
+            <img src="img/close-icon.png" alt="Close" class="close_icon">
+            <ul class="mobile_nav">
+                <a href="about.html"><li>About</li></a>
+                <a href="#"><li>Program</li></a>
+                <a href="#"><li>Join</li></a>
+                <a href="#"><li>Sponsor</li></a>
+                <a href="#"><li>News</li></a>
+            </ul>
+        </section>
```
Comment (joel-kalema) [inline about.html]: - [x] You did great work @hmunish 😃, however, in this `about.html` page we are not supposed to have again the `about` button in your navbar. What do you think if we change it to this `<li><a href="index.html">Home</a></li>` to facilitate the user to go back to the `home` page 👍? Kindly fix it and make sure it is the same for the `mobile menu` 💪.  

- [x]  Also for a good semantic syntax, kindly use this and make sure it is the same for your `index.html`:

```` bash
 <section>
            <ul>
                <li><a href=""> ... </a></li>
                <li><a href=""> ... </a></li>
                <li><a href=""> ... </a></li>
                <li><a href=""> ... </a></li>
            </ul>
  </section>
````

File: index.html [hunk 2]
```diff
@@ -5,8 +5,168 @@
...
+
+        <!-- Speakers Section Starts Here -->
+
+        <section class="speakers d-flex flex-column align-items-center justify-content-between">
+            <h3 class="feature d-flex flex-column justify-content-between align-items-center" id="speakers_title">Featured Speakers<span></span></h3>
+            <ul class="d-flex flex-sm-column flex-md-row justify-content-between align-items-center" id="speakers_container">
+                <!-- Speakers details will be loaded by javascript -->
+            </ul>
+            <button class="more_speakers mobile">More<img src="img/down-arrow.png" class="down_icon" alt="Down"></button>
+        </section>
```
Comment (joel-kalema) [inline index.html]: - [x] To make your project more responsive, Kindly display your Speakers in a column in the mobile version as on the original design 👍 
- [x] Also, Please, make sure your `button` look the same on the design 😃 

> | | Figma template | | Your project | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-05-24 140121| |Screenshot 2023-05-24 135517| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @hmunish 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @hmunish,

# STATUS: APPROVED 😃 

leonardo-dicaprio-leo-dicaprio

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 
- Professional README file ✔️ 
- Good UI and UX ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**